tasks/install-matlab/v1-beta/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can the "v1-beta" containing this file also be changed to "v1"?

Reply:
updated